### PR TITLE
Updating Unity Webgl plugin docs

### PR DESCRIPTION
**Pull request by AdhikSharma**
Added doc for unity webgl plugin
Pages
- Main Page
- Methods
- Local Peer
- Remote Peer
- Init Page
- Room

### CODE REVIEW

File: pages/UnityWebglPlugin/methods/init.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+```csharp filename=".csharp"
+using UnityEngine;
+using Huddle01;
+
+public class AudioVideoMeet : MonoBehaviour
+{
+		void Start()
+    {
+        Huddle01Init.Instance.Init("Your_project_id");
```
Comment (arcinston) [inline pages/UnityWebglPlugin/methods/init.mdx:21]: why is the name Huddle01Init? 

rename to Huddle01 or rtc everywhere

**Review by arcinston — CHANGES_REQUESTED** [review]
Walkthrough for the example app needs to be added
this is crucial considering the docs have no storyline 


Overall i do not like the way the doc is currently structured (needs more breakdown in sections ) , Flutter SDK Docs are still a decent example to refer to where structure is similar but more descriptive

not all functions have descriptions to them , lack of defination of parameters like 
image

also notes like this 
image
should also be added whereever necessary

Best Examples are still React SDK but maybe Flutter is a better reference for Unity

File: pages/UnityWebglPlugin/methods/init.mdx (hunk 1 above)
Comment (AdhikSharma) [reply pages/UnityWebglPlugin/methods/init.mdx:21]: Huddle01 is the namespace that is already defines
We can change Huddle01Init to Huddle01Core

Comment (AdhikSharma) [reply pages/UnityWebglPlugin/methods/init.mdx:21]: As HuddleCore is not exposed to change, its better not to expose any information related to it.
Unity webgl is more like a plugin which interacts with HuddleCore JS, so exposing methods with minimum information is better.

Comment (AdhikSharma) [reply pages/UnityWebglPlugin/methods/init.mdx:21]: Unity webgl plugin majorly has one main class with all major methods, events and a native c# class which interacts with JS. There is no class or component which dev can inherit or expand. 
For better clarity we can segregate methods and events.

**Pull request merged**